src/main/cli/cli.c

```diff
@@ -1314,17 +1315,26 @@ static void cliSerial(const char *cmdName, char *cmdline)
     memset(&portConfig, 0 , sizeof(portConfig));
 
     uint8_t validArgumentCount = 0;
+    int val = 0;
 
     const char *ptr = cmdline;
+    char *tok = cmdline;
+    serialPortIdentifier_e identifier = findSerialPortByName(strsep(&tok, " "), strcasecmp);
+    if (identifier == SERIAL_PORT_NONE) {
+        identifier = atoi(ptr++);
+    } else {
+        ptr = tok;
+    }
 
-    int val = atoi(ptr++);
-    serialPortConfig_t *currentConfig = serialFindPortConfigurationMutable(val);
+    serialPortConfig_t *currentConfig = serialFindPortConfigurationMutable(identifier);
```

Comment:
`serialFindPortConfigurationMutable` should be fixed to return NULL for SERIAL_PORT_NONE

Reply:
Already done :)